src/ontology/Makefile

```diff
@@ -58,10 +65,10 @@ RELEASE_ARTEFACTS = $(sort $(ONT)-base $(ONT)-full $(ONT)-simple $(ONT)-basic ex
 .PHONY: .FORCE
 
 .PHONY: all
-all: odkversion all_imports all_main all_subsets sparql_test all_reports all_assets
+all: odkversion all_imports patterns all_main all_subsets sparql_test all_reports all_assets
 
 .PHONY: test
-test: odkversion sparql_test all_reports 
+test: odkversion pattern_schema_checks sparql_test all_reports 
```

Comment:
@dosumis This is still your hack. On the ODK side, the fix to enable pattern validation when `use_dosdps` is set to True is only present in the master branch, not in any released version.

You can leave your hack as it is. At the next ODK release, as soon as someone does `update_repo`, the Makefile will be overwritten by the ODK-generated one, which will contain all the necessary bits to perform pattern validation.

Reply:
dont think this is standard, I've check PCL (which is a super clean odk makefile) and it isn't there

Reply:
I think OK to merge as it doesn't (currently!) overload.  I know @matentzn is busy refactoring pattern checking.  I would like to make sure basic pattern schema checks happen every time and early in testing though.  @matentzn is there some documentation on the new order of testing for patterns?

Reply:
The order is defined by the `test:` goal in the makefile, and I have now moved DOSDP validation as the first step in the order, because it is so fast. After that, it is
- custom sparql
- robot reports
- OWL DL profile validation
- Reasoning (including unsat and equivalent class check).

In the default set up.